### PR TITLE
Adds new and improved flux screaming syndrome.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
add: Added a horrible new disease that causes victims to "oof" themselves to death, this disease randomizes its cure in its early stages. 
/:cl:

[why]: THE PEOPLE HATH SPOKEN!

unknown

image

This disease is adminspawn only(for now) and an utter meme but hey people asked for it.

Everything is tested and works nicely, it changes your scream during infection and resets it upon cure. 

Near the end it spams you with oxyloss as you cannot scream while unconscious.

### CODE REVIEW

**Comment by DinduStuffin** [comment]
+1
 +1 a million times

File: hippiestation/code/datums/diseases/fluxitus.dm [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	name = "Fluctuating Screaming Syndrome"
+	max_stages = 5
+	stage_prob = 5 //Faster than normal
+	spread_text = "Airborne"
+	cure_text = "Constantly changing"
+	cures = list("clf3") //KILL IT WITH FIRE!! this will change early in the diseases lifespan but if some madman is quick with the hot stuff they get to be cured.
+	agent = "Brain damaging prions"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	cure_chance = 15
+	desc = "This disease constantly changes and shifts in the host and causes a build-up of oxygen depriving chemicals that damage the brain and cause the body to reflexively scream in a desperate attempt to gain more oxygen."
```
Comment (KayeArray) [inline hippiestation/code/datums/diseases/fluxitus.dm:11]: that's a long sentence

File: hippiestation/code/datums/diseases/fluxitus.dm [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	severity = VIRUS_SEVERITY_BIOHAZARD
+	var/new_scream = list('hippiestation/sound/misc/oof.ogg')
+	var/alternate_cures = list("mannitol", "synaptizine", "cryoxadone", "salbutamol", "perfluorodecalin", "morphine", "oculine", "epinephrine", "mutadone", "antihol", "gold")
+	var/new_cure = list("mannitol")
```
Comment (KayeArray) [inline hippiestation/code/datums/diseases/fluxitus.dm:15]: why is this a list?

**Comment by ghost** [comment]
+1

File: hippiestation/code/datums/diseases/fluxitus.dm (hunk 2 above)
Comment (GuyonBroadway) [reply hippiestation/code/datums/diseases/fluxitus.dm:15]: Cures for diseases are always lists, it does some list length operations and runtimes if ita not a list.

Comment (KayeArray) [reply hippiestation/code/datums/diseases/fluxitus.dm:15]: but you're just adding it to the regular cures list

**Review by ghost — APPROVED** [review]

**Comment by PopNotes** [comment]
+1 do it right meow

**Comment by doobiedoo23** [comment]
YE

**Comment by KayeArray** [comment]
this breaks you will kill you

**Pull request merged**